hudi-hive/src/main/java/org/apache/hudi/hive/HoodieHiveClient.java

```diff
@@ -116,6 +126,19 @@ public HoodieHiveClient(HiveSyncConfig cfg, HiveConf configuration, FileSystem f
     activeTimeline = metaClient.getActiveTimeline().getCommitsTimeline().filterCompletedInstants();
   }
 
+  private static void closeQuietly(ResultSet resultSet, Statement stmt) {
+    try {
+      if (stmt != null) {
+        stmt.close();
+      }
+      if (resultSet != null) {
+        resultSet.close();
+      }
+    } catch (SQLException e) {
```

Comment:
If stmt.close() throws SQLException. the resultSet would not be closed. Could it be the following?
```
try {
      if (stmt != null) {
        stmt.close();
      }
} catch (SQLException e) {
	// log
}

try {
      if (resultSet != null) {
        resultSet.close();
      }
} catch (SQLException e) {
	// log
}
```